rvc/lib/tools/pretrained_selector.py

```diff
@@ -1,63 +1,14 @@
-def pretrained_selector(pitch_guidance):
-    if pitch_guidance == True:
-        return {
-            "v1": {
-                32000: (
-                    "rvc/models/pretraineds/pretrained_v1/f0G32k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/f0D32k.pth",
-                ),
-                40000: (
-                    "rvc/models/pretraineds/pretrained_v1/f0G40k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/f0D40k.pth",
-                ),
-                48000: (
-                    "rvc/models/pretraineds/pretrained_v1/f0G48k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/f0D48k.pth",
-                ),
-            },
-            "v2": {
-                32000: (
-                    "rvc/models/pretraineds/pretrained_v2/f0G32k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/f0D32k.pth",
-                ),
-                40000: (
-                    "rvc/models/pretraineds/pretrained_v2/f0G40k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/f0D40k.pth",
-                ),
-                48000: (
-                    "rvc/models/pretraineds/pretrained_v2/f0G48k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/f0D48k.pth",
-                ),
-            },
-        }
-    elif pitch_guidance == False:
-        return {
-            "v1": {
-                32000: (
-                    "rvc/models/pretraineds/pretrained_v1/G32k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/D32k.pth",
-                ),
-                40000: (
-                    "rvc/models/pretraineds/pretrained_v1/G40k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/D40k.pth",
-                ),
-                48000: (
-                    "rvc/models/pretraineds/pretrained_v1/G48k.pth",
-                    "rvc/models/pretraineds/pretrained_v1/D48k.pth",
-                ),
-            },
-            "v2": {
-                32000: (
-                    "rvc/models/pretraineds/pretrained_v2/G32k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/D32k.pth",
-                ),
-                40000: (
-                    "rvc/models/pretraineds/pretrained_v2/G40k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/D40k.pth",
-                ),
-                48000: (
-                    "rvc/models/pretraineds/pretrained_v2/G48k.pth",
-                    "rvc/models/pretraineds/pretrained_v2/D48k.pth",
-                ),
-            },
-        }
+def pretrained_selector(version, vocoder, pitch_guidance, sample_rate):
+    
+    path = f"rvc/models/pretraineds/pretrained_{version}/"
+    f0 = "f0" if pitch_guidance == True else ""
+
+    if vocoder == "default":
+        vocoder_path = ""
+    elif vocoder == "MRF HiFi-GAN":
+        vocoder_path = "HiFiGAN_"
+
+    path_g = f"{path}{vocoder_path}{f0}G{str(sample_rate)[:2]}k.pth"
```

Comment:
This requires a file structure change, right?

Reply:
No, just need to drop a new pretrain file into the existing folder.